.github/workflows/linux-tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false 
       matrix:
-        PYTHON_VERSION: ['2.7', '3.6', '3.7', '3.8']
```

Comment:
I'd like to maintain compatibility with Python 2 for a little longer here so people can still evaluate their Python 2 code. Since that doesn't represent a big effort for this small package, please revert these changes.

Reply:
Done